shared/video-sdk/_get-started-uikit.mdx

```diff
@@ -31,7 +31,7 @@ To use <Vg k="UIK" />, you implement the following steps in your app:
 
 2.  **Join a channel**
 
-    When you execute the `join` command, your <Vpl k="CLIENT" /> joins the channel, publishes the local audio and video streams to <Vg k="AGORA_BACKEND" />, and handles audio and video for anyone else who joins the call in a default UI.
+    When you execute the `join` command, your <Vpl k="CLIENT" /> joins the channel, publishes the local audio and video streams to [<Vg k="AGORA_BACKEND" />](../overview/core-concepts#agora-sd-rtn), and handles audio and video for anyone else who joins the call in a default UI.
```

Comment:
Please double check this link in a running server to ensure it is good.

Reply:
Checked. The link is working